src/System.Device.Gpio/Interop/Unix/libbcm_host/Interop.libbcmhost.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class libbcmhost
+    {
+        private const string LibbcmhostLibrary = "libbcm_host";
```

Comment:
is this available on every PI? (I can see it on my Pi3 under `/opt/vc/lib/libbcm_host.so` but not sure about Pi2/Pi4). Also does this need `.so` extension? (If you hit this code path and works fine then presumably works)

Reply:
No it's not available on every pi (see comment later on).  but when it is available then It is reccomended that it is used by the Pi folk. As you say it works without the .so suffix and I note that the other interrop libraries don't use a suffix either.